cmd/docker-driver/driver.go

```diff
@@ -101,9 +121,16 @@ func (d *driver) StopLogging(file string) {
 	level.Debug(d.logger).Log("msg", "Stop logging", "file", file)
 	d.mu.Lock()
 	lf, ok := d.logs[file]
-	if ok {
-		lf.Close()
-		delete(d.logs, file)
+	if !ok {
+		return
```

Comment:
This return would bypass `d.mu.Unlock()`, is that OK?

Reply:
You're probably right ! good catch.